media-video/vlc/vlc-3.0.16.recipe

```diff
@@ -62,13 +70,14 @@ REQUIRES="
 	lib:libQt5Widgets$secondaryArchSuffix
 	lib:libsamplerate$secondaryArchSuffix
 	lib:libschroedinger_1.0$secondaryArchSuffix
+	lib:libsmb2$secondaryArchSuffix 
 	lib:libspeex$secondaryArchSuffix
 	lib:libswscale$secondaryArchSuffix
 	lib:libtag$secondaryArchSuffix
 	lib:libtheoradec$secondaryArchSuffix
 	lib:libtheoraenc$secondaryArchSuffix
 	lib:libtwolame$secondaryArchSuffix
-#	lib:libupnp$secondaryArchSuffix					# deadlock during upnp devices scan
+	lib:libupnp$secondaryArchSuffix					# deadlock during upnp devices scan
```

Comment:
Does it not deadlock anymore?

Reply:
It doesn't, anymore. Butit doesn't means it works.